docs/SOPs/contributor_communication_SOP.md

```diff
@@ -146,6 +152,36 @@ These templates are considered a guide to provide a skeleton with the required i
 > 
 > On behalf of the HCA DCP Data Wrangling Team
 
+### HCA branded publication - no preprint or publication 
+
+> Dear [name/s of contributor/s],
+> 
+> Congratulations on your HCA manuscript! We are looking forward to hearing more about your project, and will add your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication once a preprint or publication is available. 
+> 
+> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process.
+> 
+> Warm regards,
+> [wrangler name] 
+> 
+> On Behalf of the HCA DCP Ingestion Service
+
+#### HCA branded publication - with preprint or publication
+
+> Dear [name/s of contributor/s],
+> 
+> Congratulations on your HCA manuscript!
+> We have added the information about your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication.
+> Due to internal processes it will take a day to display the publication information, in the meantime any feedback about the HCA project catalogue is greatly appreciated!
+> 
+> If you have already archived your data in any database, could you provide us with the accession number?
+> 
+> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also in that case, please inform us of any time-sensitive requirements on archiving.
```

Comment:
```suggestion
> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also in that case, please inform us of any time-sensitive requirements.
```

Reply:
If they already deposited their data in an archive there is no need for us to offer archiving. 
We are unable to archive right now anyway so there is no point in mentioning it